### PR TITLE
Configure the app to use postgres database.

### PR DESCRIPTION
**Pull request by TuyizeeAnastase**
**What does this PR do?**
Sequelizing postgres database

**Tasks to be completed**
-installing sequelize package and initialize them
-create database 
-migrate and seed the tables
-check if data saved in database

**How can be manually tested?**
-clone the repo
-npm install
-create database called test
-create .env file and save here database string as DEV_DATABASE_URL
-npm migrate
-npm run seed

**What are relavant trello story**
https://trello.com/c/3RYihb3M/17-configure-the-app-to-use-postgres-database

### CODE REVIEW

**Review by Ntwali-Josue — APPROVED** [review]

**Review by Ntwali-Josue — APPROVED** [review]

**Review by Ntwali-Josue — CHANGES_REQUESTED** [review]
solve the conflicts you're having in the files that github is showing you

**Comment by TuyizeeAnastase** [comment]
yeah, Good resolve those conflicts and trevis error.

**Review by Mugema123 — COMMENTED** [review]
you can squash and remain with one commit. but first remove those conflicts shown github.

**Comment by TuyizeeAnastase** [comment]
trevis error checked and conflicts resolved.

File: .sequelizerc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const path = require('path')
+
+module.exports = {
+  config: path.resolve('./database/config', 'config.js'),
+  'models-path': path.resolve('./database/models'),
+  'seeders-path': path.resolve('./database/seeders'),
+  'migrations-path': path.resolve('./database/migrations'),
+}
```
Comment (Ntwali-Josue) [inline .sequelizerc]: add a new line at the end of the file

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "git.defaultCloneDirectory": ""
+}
```
Comment (Ntwali-Josue) [inline .vscode/settings.json]: add a new line at the end of the file

File: database/config/config.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  },
+  test: {
+    url: process.env.TEST_DATABASE_URL,
+    dialect: 'postgres',
+  },
+  production: {
+    url: process.env.DATABASE_URL,
+    dialect: 'postgres',
+  },
+}
```
Comment (Ntwali-Josue) [inline database/config/config.js]: add a new line at the end of the file

File: database/models/user.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  User.init({
+    name: DataTypes.STRING,
+    email: DataTypes.STRING,
+    phone: DataTypes.STRING
+  }, {
+    sequelize,
+    modelName: 'User',
+  });
+  return User;
+};
```
Comment (Ntwali-Josue) [inline database/models/user.js]: add a new line at the end of the file

File: database/migrations/20200922194501-create-user.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
+  },
+  down: async (queryInterface, Sequelize) => {
+    await queryInterface.dropTable('Users');
+  }
+};
```
Comment (Ntwali-Josue) [inline database/migrations/20200922194501-create-user.js]: add a new line at the end of the file

File: .sequelizerc (hunk 1 above)
Comment (TuyizeeAnastase) [reply .sequelizerc]: Done.

File: .vscode/settings.json (hunk 2 above)
Comment (TuyizeeAnastase) [reply .vscode/settings.json]: Done

File: database/config/config.js (hunk 3 above)
Comment (TuyizeeAnastase) [reply database/config/config.js]: Done.

File: database/migrations/20200922194501-create-user.js (hunk 5 above)
Comment (TuyizeeAnastase) [reply database/migrations/20200922194501-create-user.js]: Done

File: database/models/user.js (hunk 4 above)
Comment (TuyizeeAnastase) [reply database/models/user.js]: Done

**Review by Mugema123 — APPROVED** [review]

**Review by Ntwali-Josue — APPROVED** [review]

File: test/test.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
+import app from "../server/app";
+import chai from "chai";
+import chaiHttp from "chai-http";
+import config from "../database/config/config";
+import migration from "../database/migrations/20200922194501-create-user";
+import modalIndex from "../database/models/index";
+import modalUser from "../database/models/user";
+import seedUser from "../database/seeders/20200922194759-User"
```
Comment (blaise82) [inline test/test.js]: why is this imported here?

**Review by blaise82 — CHANGES_REQUESTED** [review]
remove all seeding files
or any migrations all we need is a database setup

File: test/test.js (hunk 6 above)
Comment (TuyizeeAnastase) [reply test/test.js]: removed

**Review by Ally4 — APPROVED** [review]

**Review by Ntwali-Josue — APPROVED** [review]

**Review by blaise82 — APPROVED** [review]

**Pull request merged**